hawk/app/controllers/cib_controller.rb

```diff
@@ -7,6 +7,15 @@ class CibController < ApplicationController
 
   def show
     respond_to do |format|
+      format.html do
+        out, err, rc  = Invoker.instance.crm_configure "show"
```

Comment:
Lint/UselessAssignment: Useless assignment to variable - `err`. Use `_` or `_err` as a variable name to indicate that it won't be used.<br>Style/ExtraSpacing: Unnecessary spacing detected.